polylogue/cli/maintain.py

```
@@ -20,6 +24,42 @@ def run_maintain_cli(args: argparse.Namespace, env: CommandEnv) -> None:
         run_doctor_cli(args, env)
     elif maintain_cmd == "index":
         run_index_cli(args, env)
+    elif maintain_cmd == "restore":
+        src: Path = Path(getattr(args, "src"))
```

Ruff (0.15.9)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)